atc/creds/cached_secrets.go

```diff
@@ -36,16 +37,16 @@ func NewCachedSecrets(secrets Secrets, cacheConfig SecretCacheConfig) *CachedSec
 	}
 }
 
-func (cs *CachedSecrets) Get(secretPath string) (interface{}, *time.Time, bool, error) {
+func (cs *CachedSecrets) Get(ref vars.VariableReference) (interface{}, *time.Time, bool, error) {
```

Comment:
What's the reason that all of the credential managers take in a `VariableReference`, rather than just the `Path`? They don't seem to consume the `Source` or `Fields` from what I can tell

Reply:
Guess we could still pass `path` as the argument.